src/App.js

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import './Button.css';
 
+import PokemonCard from "./components/pokemonCard/PokemonCard";
+
+import axios from "axios";
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
-}
+    const [pokemonArray, setPokemonArray] = useState([]);
+    const [arrayMetaData, setArrayMetaData] = useState({});
```

Comment:
Ik zou hiervan de default waarde `undefined` maken zodat je een goed onderscheid hebt tussen of het al wél of niet ingeladen is
```suggestion
    const [arrayMetaData, setArrayMetaData] = useState();
```